Xext/panoramiX.c

```diff
@@ -763,7 +765,7 @@ PanoramiXMaybeAddVisual(VisualPtr pVisual)
 
             if ((*XineramaVisualsEqualPtr) (pVisual, walkScreen, candidate)
 #ifdef GLXPROXY
-                && glxMatchVisual(screenInfo.screens[0], pVisual, pScreen)
+                && glxMatchVisual(screenInfo.screens[0], pVisual, walkScreen)
```

Comment:
Huh, how did it compile before (with `GLXPROXY` defined)? `pScreen` was not defined.

Reply:
did a little digging ... turned out that this symbol hasn't been defined for at least a decade - these lines had been introduced with DMX, decades ago, which had been removed aeons ago.
I'll send a separate PR for removing this relic.